src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java

```diff
@@ -454,6 +465,48 @@ public Encounter(long time, String type) {
       devices = new ArrayList<Device>();
       this.claim = new Claim(this, person);
     }
+
+    /**
+     * Add an observation to the encounter.
+     * @param time The time of the observation
+     * @param type The LOINC code for the observation
+     * @param value The observation value
+     */
+    public void addObservation(long time, String type, Object value) {
+      Observation observation = new Observation(time, type, value);
+      observation.codes.add(new Code("LOINC", type, type));
+      this.observations.add(observation);
+    }
+
+    /**
+     * Find the first observation in the encounter with the given LOINC code.
+     * @param code The LOINC code to look for
+     * @return A single observation or null
+     */
+    public Observation findObservation(String code) {
+      return observations
+          .stream()
+          .filter(o -> o.containsCode(code, "LOINC"))
+          .findFirst()
+          .orElse(null);
+    }
```

Comment:
This also seems duplicative of the `public Observation getLatestObservation(String type)` method.

Reply:
Same comment as above.